model/rpc/spans.yaml

```diff
@@ -1,8 +1,8 @@
 groups:
   - id: rpc
```

Comment:
this is one of the several reasons we can't ~~have  nice things~~ generate span kind in MD - we define spans without kind.

TODO: create issue to require kind and clean up stuff.